src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.scss

```diff
@@ -0,0 +1,28 @@
+.repository-button {
+    margin-bottom: 10px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+}
+
+.remove-line-left {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-left: none;
+}
+
+.remove-line-right {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-right: none;
+}
+
+.remove-line-left-right {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-left: none;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
```

Comment:
You can replace all `border-...-...-radius` with one single `border-radius` as this will cover all cases.

Reply:
Thanks for the hint 👍